pulp_container/tests/functional/utils.py

```diff
@@ -112,6 +111,22 @@ def gen_container_client():
     return ContainerApiClient(configuration)
 
 
+def gen_repo(**kwargs):
+    """Return a semi-random dict for use in creating a Repository."""
+    data = {"name": str(uuid4())}
+    data.update(kwargs)
+    return data
+
+
+def gen_remote(url, **kwargs):
+    """Return a semi-random dict for use in creating a Remote.
+    :param url: The URL of an external content source.
+    """
+    data = {"name": str(uuid4()), "url": url}
+    data.update(kwargs)
+    return data
+
+
 def gen_container_remote(url=REGISTRY_V2_FEED_URL, **kwargs):
```

Comment:
Are we still using these? I'd like to see them being merged into the factory fixtures. But maybe not today.

Reply:
Don't leave for tomorrow what you can break today! :smile: